widget-template/build.html

```diff
@@ -1,4 +1,4 @@
-<div class="container" data-{{safeName}}-id="{{id}}">
+<div class="container" data-{{safeName}}-1-0-id="{{id}}">
```

Comment:
If the [generated widget.json](https://github.com/Fliplet/fliplet-cli/blob/c7c29d500e49fda31c8ebe47886b36db1d28a016/widget-template/widget.json#L4) uses `1.0.0` as the initial version, this should be `1-0-0`?

Reply:
@tonytlwu I updated to use `1-0-0`

Reply:
So extreme :trollface: I prefer `one-zero-zero`

Reply:
Let's go with `1-0-0`. I kinda hate it, but this way there's no mistaking that it means version. Otherwise a `-1` could mean various things.